Lucheti29/Training
# News

## Summary

News layout implemented with features like toolbar (tabs), ListView, FloatingActionButton, SwipeRefreshLayout and RoundedImageView.
## Test
- Click on tabs to see how the layout changes.
- Pull down and pull up to check how the list view works.  
- When you stay on first list element, pull down and see the SwipeRefreshLayout (for now, it loads nothing).
## Screenshot

![News Layout](http://s8.postimg.org/5dbag6lud/news.png)
## Card

https://trello.com/c/40sP1lem/16-news
